ui/app/AppLayouts/Wallet/popups/swap/SwapModalAdaptor.qml

```diff
@@ -189,4 +223,37 @@ QObject {
         }
         return totalBalance
     }
+
```

Comment:
this new function have gone to internal properties and function section can you please move it on top?

Reply:
Sorry I don't understand... I saw we normally do internal stuff first and public stuff at the bottom, is that not the case?

Reply:
not sure, was done this way in this file so far. but in any case lets just make it consistent?